create-publication-from-scopus/src/test/java/no/sikt/nva/scopus/ScopusHandlerTest.java

```diff
@@ -35,6 +35,8 @@
 import no.sikt.nva.scopus.test.utils.ScopusGenerator;
 import no.unit.nva.metadata.CreatePublicationRequest;
```

Comment:
It appears it is the same test that is also failing on main branch that prevents building